pkg/serviceaccounts/oauthclient/oauthclientregistry_test.go

```
@@ -1005,7 +1042,8 @@ func TestGetRedirectURIs(t *testing.T) {
 		},
 	} {
 		a := buildRouteClient(test.routes)
-		actual := test.models.getRedirectURIs(a.redirectURIsFromRoutes(test.namespace, test.models.getNames()))
+		uris, _ := a.redirectURIsFromRoutes(test.namespace, test.models.getNames())
```

Don't ignore the errors.